packages/babel-core/test/option-manager.js

```diff
@@ -160,23 +160,21 @@ describe("option-manager", () => {
           ],
         });
 
-        assert.equal(true, Array.isArray(options.plugins));
-        assert.equal(1, options.plugins.length);
-        assert.equal(0, options.presets.length);
+        expect(true).toBe(Array.isArray(options.plugins));
+        expect(1).toBe(options.plugins.length);
+        expect(0).toBe(options.presets.length);
```

Comment:
Can you reverse those conditions? (And use the appropriate `expect().*` functions)

Sorry for not spotting this in my earlier review.

Reply:
Added these changes. Thanks for pointing out.